common-ts/src/utils/math.ts

```diff
@@ -0,0 +1,74 @@
+import { BN, L2OrderBook, PERCENTAGE_PRECISION } from '@drift-labs/sdk';
+
+const calculatemarkPrice = (bestBidPrice: BN, bestAskPrice: BN) => {
+	return bestBidPrice.add(bestAskPrice).divn(2);
```

Comment:
do you know if this is what we were using before? @0xbigz you should review this, not sure what you had when it was crossing before